docs/kubernetes-quickstart.md

```diff
@@ -0,0 +1,114 @@
+# Quickstart
```

Comment:
Instead of having a whole other quickstart doc, maybe we should combine this into the quickstart doc I started. We can probably update the quickstart.sh script to use the commands you have below and combine our efforts. That way the user can run the shell script to startup the containers instead of the localized applications. Thoughts?

Reply:
yeah i think that would make sense. Lemme rebase and combine the stuff here into the other quickstart